docs/components/store.md

```diff
@@ -212,6 +212,7 @@ config:
   max_idle_connections: 0
   max_async_concurrency: 0
   max_async_buffer_size: 0
+  max_item_size: 0
```

Comment:
Maybe let's recommend users to set this to `1MiB` as that is the default memcached's limit instead of disabling it?

Reply:
Would you just change this example config or the default value in the code?

Reply:
What do you mean by **or** in your sentence? We could add `omitempty` on `MaxItemSize` & change the value in `defaultMemcachedClientConfig` to `1MiB` plus change it here to be `1MiB`. I think our users would be better off if we would recommend sane values with the default memcached configuration :-)

Reply:
Makes sense to me. I've changed the default from `0` to `1MB` (which is the default in Memcached). What's your take?

Reply:
My take is that this is awesome :+1: